{"hierarchy":{"paths":[["doc:\/\/com.cloudx.sdk.core\/documentation\/CloudXCore","doc:\/\/com.cloudx.sdk.core\/documentation\/CloudXCore\/CloudXError"]]},"schemaVersion":{"patch":0,"minor":3,"major":0},"identifier":{"url":"doc:\/\/com.cloudx.sdk.core\/documentation\/CloudXCore\/CloudXError\/nativeViewError","interfaceLanguage":"swift"},"primaryContentSections":[{"declarations":[{"platforms":["iOS"],"tokens":[{"text":"case","kind":"keyword"},{"text":" ","kind":"text"},{"kind":"identifier","text":"nativeViewError"}],"languages":["swift"]}],"kind":"declarations"}],"metadata":{"roleHeading":"Case","externalID":"s:10CloudXCore0A6XErrorO15nativeViewErroryA2CmF","symbolKind":"case","title":"CloudXError.nativeViewError","modules":[{"name":"CloudXCore"}],"role":"symbol","fragments":[{"kind":"keyword","text":"case"},{"text":" ","kind":"text"},{"text":"nativeViewError","kind":"identifier"}]},"sections":[],"kind":"symbol","abstract":[{"text":"Something went wrong with the native view.","type":"text"}],"variants":[{"traits":[{"interfaceLanguage":"swift"}],"paths":["\/documentation\/cloudxcore\/cloudxerror\/nativeviewerror"]}],"references":{"doc://com.cloudx.sdk.core/documentation/CloudXCore":{"role":"collection","kind":"symbol","title":"CloudXCore","type":"topic","identifier":"doc:\/\/com.cloudx.sdk.core\/documentation\/CloudXCore","abstract":[],"url":"\/documentation\/cloudxcore"},"doc://com.cloudx.sdk.core/documentation/CloudXCore/CloudXError":{"title":"CloudXError","fragments":[{"kind":"keyword","text":"enum"},{"kind":"text","text":" "},{"text":"CloudXError","kind":"identifier"}],"identifier":"doc:\/\/com.cloudx.sdk.core\/documentation\/CloudXCore\/CloudXError","url":"\/documentation\/cloudxcore\/cloudxerror","kind":"symbol","role":"symbol","type":"topic","navigatorTitle":[{"text":"CloudXError","kind":"identifier"}],"abstract":[{"type":"text","text":"An error type that represents the CloudX SDK errors."}]},"doc://com.cloudx.sdk.core/documentation/CloudXCore/CloudXError/nativeViewError":{"role":"symbol","kind":"symbol","title":"CloudXError.nativeViewError","identifier":"doc:\/\/com.cloudx.sdk.core\/documentation\/CloudXCore\/CloudXError\/nativeViewError","type":"topic","url":"\/documentation\/cloudxcore\/cloudxerror\/nativeviewerror","abstract":[{"text":"Something went wrong with the native view.","type":"text"}],"fragments":[{"text":"case","kind":"keyword"},{"text":" ","kind":"text"},{"kind":"identifier","text":"nativeViewError"}]}}}